src/main/java/org/jenkinsci/plugins/p4/console/P4ConsoleAnnotator.java

```diff
@@ -34,6 +34,22 @@ private void push(MarkupText text) {
 		text.addMarkup(COMMAND.length(), head);
 
 		StringBuffer sb = new StringBuffer();
+
+		sb.append("<script>\n");
+		sb.append("function toggle(showHideDiv, switchTextDiv) {\n");
+		sb.append("\tvar tog = document.getElementById(showHideDiv);\n");
+		sb.append("\tvar text = document.getElementById(switchTextDiv);\n");
+		sb.append("\tif(tog.style.display == \"block\") {\n");
+		sb.append("    \ttog.style.display = \"none\";\n");
+		sb.append("\t\ttext.innerHTML = \"+\";\n");
+		sb.append("  \t}\n");
+		sb.append("\telse {\n");
+		sb.append("\t\ttog.style.display = \"block\";\n");
+		sb.append("\t\ttext.innerHTML = \"-\";\n");
+		sb.append("\t}\n");
+		sb.append("}\n");
+		sb.append("</script>");
+
```

Comment:
The toggle (+) works fine in the page "Console Output" (e.g. http://localhost:8080/job/Test_Pipeline_P4_Sync/2/console) because of resources/org/jenkinsci/plugins/p4/console/P4ConsoleNote/script.js. But for both the page "Perforce Software Polling Log" (e.g. http://localhost:8080/job/Test_Pipeline_P4_Sync/scmPollLog) and the page "Polling Log" (e.g. http://10.202.11.14:8080/job/Test_Pipeline_P4_Sync/2/pollingLog) can't work. Because those two pages can't include the script.js (which have the toggle function).
So that is why I append the javascript function in its log to cope with this issue. I don't find better way to overload the jelly configuration for hudson.triggers.SCMTrigger.SCMAction and hudson.triggers.SCMTrigger.BuildAction. Because they don't like Console Output which has the mechanism to let add customer's script.js:
-> hudson/model/Run/console.jelly
-> <j:out value="${h.generateConsoleAnnotationScriptAndStylesheet()}"/>
-> hudson.Functions#generateConsoleAnnotationScriptAndStylesheet